src/Symfony/Component/Scheduler/Trigger/TriggerInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Scheduler\Trigger;
+
+interface TriggerInterface
+{
+    public function nextTo(\DateTimeImmutable $run): ?\DateTimeImmutable;
```

Comment:
What about just `next`?

Reply:
It was: `getNextRun`, `nextRun`, `next`. Then my colleague asked me why I am set a time here... Now it is `nextTo($run)` that sounds clearer to me. I'd like to name it better if possible.

Reply:
CronExpression names it `getNextRunDate()`, which sounds very descriptive to me.